openzaak/openzaak-database/database/fill-data-on-startup.sh

```diff
@@ -0,0 +1,28 @@
+# SPDX-FileCopyrightText: 2025 INFO.nl
+# SPDX-License-Identifier: EUPL-1.2+
+#!/bin/bash
+
+echo ">>>>  Waiting until Open Zaak has initialized the database <<<<"
+useradd openzaak
+while true
+do
+    verifier=$(psql -U openzaak -d openzaak -t -A -c "select count(id) from accounts_user where username = 'admin'")
+    if [ "1" = "$verifier" ]
+        then
+            echo "Running database setup scripts ..."
+            for file in /docker-entrypoint-initdb.d/database/*.sql
+            do
+                echo "Running $file ..."
+                psql -U openzaak openzaak \
+                    -v BAG_API_CLIENT_MP_REST_URL="${BAG_API_CLIENT_MP_REST_URL}" \
```

Comment:
This is for the integration from OpenZaak to BAG. Is that needed here? Do we set `BAG_API_CLIENT_MP_REST_URL` even?